server/storage.go

```diff
@@ -337,7 +337,7 @@ func NewGDriveStorage(clientJSONFilepath string, localConfigPath string, basedir
 		return nil, err
 	}
 
-	srv, err := drive.New(getGDriveClient(config, localConfigPath, logger))
+	srv, err := drive.New(getGDriveClient(config, localConfigPath, logger)) // nolint: staticcheck
```

Comment:
why this is needed?

Reply:
It is needed because the drive.New function is deprecated and would need some work to update it.
Should i add a ToDo on top?